up-l1/zenoh.adoc

```diff
@@ -86,82 +96,107 @@ Different message types **MUST** use different Zenoh API.
 | CS6 | REAL_TIME
 |===
 
-=== URI Mapping
+==== TTL
 
-UUri **MUST** be able to be serialized into link:../basics/uri.adoc#42-micro-uris[micro form].
-Then we can transform the micro-form UUri, which is `u8` array, into string in lowercase hexadecimal representation.
+The TTL only takes effect while sending RPC in the Zenoh implementation.
+TTL **SHOULD** be mapped to the timeout configuration in Zenoh query.
 
-Zenoh key for uProtocol consists of three parts and they are separated by `/`.
-Zenoh key **MUST** start with `upl/` or `upr/`.
-While the local UUri maps to `upl/`, the remote UUri maps to `upr/`.
-If UAuthority is defined in UUri, it **MUST** be put in the second part.
-Then the rest of the UUri **MUST** be put in the last part.
+=== URI Mapping
+
+UUri is transformed into Zenoh key in the implementation.
+The format of Zenoh key **MUST** be `up/[src authority]/[dst authority]/[entity_id]/[ue_version_major]/[resource_id]`
 
-To be more specific:
+* `up/`: Zenoh key **MUST** always start with `up/`, which can be used to identify the uProtocol traffic.
+* `src authority`: Indicates where the message comes from.
+* `dst authority`: Indicates where the message is sent to. Must be `empty` in Publish type communication. `empty` is reserved word.
+* `entity_id`, `ue_version_major` and `resource_id`: Might come from source or destination UUri depending on the communication type. They *MUST* be mapped to the upper-case base16 encoding.
 
-* The local UUri without UAuthority: `upl/[hexadecimal representation of the UUri]`
-* The remote UUri with UAuthority: `upr/[hexadecimal representation of the UAuthority]/[hexadecimal representation of the rest of the UUri]`
+[%autowidth]
+|===
+| Message Type | Zenoh Key
 
-Note that there is a special case. If the UUri only contains UAuthority with empty UEntity and UResource,
-it's defined as a special form of Zenoh key `upr/[byte hexadecimal of UAuthority]/**`.
-It can listen to all the messages sending to a certain UAuthority.
+| Publish | up/src-authority/empty/src-entity/src-version/src-resource
+| Notification | up/src-authority/dst-authority/dst-entity/dst-version/dst-resource
+| Reqest | up/src-authority/dst-authority/dst-entity/dst-version/dst-resource
+| Response | Don't need it since relying on Zenoh queryable mechanism
+|===
 
 Some examples:
```

Comment:
@evshary , I would just map the `URI notation` (a.k.a short form) to zenoh key expressions in the above table per message type in lieu of the code below to make it simpler to read

Reply:
Agree. Let me update it.